src/localizations/en.json

```
@@ -132,7 +132,7 @@
     "access_management": "Access management",
     "users": "Users",
     "reportees": "Access to other companies",
```

```suggestion
    "reportees": "External accesses",
```

Vet ikke helt hva vi skal kalle dette på engelsk ("Access on behalf of others" blir f.eks. litt langt kanskje), men kanskje dette funker? 🤷‍♀️
(Vi burde ihvertfall ikke bruke "other companies" når dette menyelementet skal vises også på privatpersoners side)